internal/commands/root.go

```diff
@@ -55,6 +57,26 @@ func muteDockerCli(dockerCli command.Cli) func() {
 	}
 }
 
+func prepareStores(dockerCli command.Cli) (store.BundleStore, store.ClaimStore, store.CredentialStore, error) {
+	appstore, err := store.NewApplicationStore(config.Dir())
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	claimStore, err := appstore.ClaimStore(dockerCli.CurrentContext())
```

Comment:
claim / credentialset should be based on the target context (not the cli context).
e.g.: I can use my local daemon to publish to production and staging cluster, and I want, credentials/claims to be isolated by target context

Reply:
of course 😓 Fixing it now